### PR TITLE
Updated to clang-format 8.0.0

### PR DESCRIPTION
**Pull request by zwaldowski**
clang-format 7.0 cannot be installed using Homebrew anymore, so upgrading is necessary for contributors to easily meet the rules in [Contributing](https://github.com/git-up/GitUp/blob/master/CONTRIBUTING.md).

I AGREE TO THE GITUP CONTRIBUTOR LICENSE AGREEMENT

### CODE REVIEW

File: GitUpKit/Core/GCFunctions.m [hunk 1]
```diff
@@ -95,7 +95,7 @@ BOOL GCGitOIDFromSHA1Prefix(NSString* prefix, git_oid* oid, NSError** error) {
 }
 
 NSString* GCUserFromSignature(const git_signature* signature) {
-  return [NSString stringWithFormat:@"%@ <%s>", [NSString stringWithUTF8String:signature->name], signature->email];
+  return [NSString stringWithFormat:@"%@ <%s>", [NSString stringWithUTF8String:signature->name], signature -> email];
```
Comment (lucasderraugh) [inline GitUpKit/Core/GCFunctions.m]: I'm confused by this one. We have 2 different styles on this same line. I'd prefer the previous style.

File: GitUpKit/Views/GIMapViewController+Operations.m [hunk 2]
```diff
@@ -622,8 +622,8 @@ - (void)mergeCommitOrBranch:(id)commitOrBranch intoLocalBranch:(GCHistoryLocalBr
                                 [self.repository setUndoActionName:[NSString stringWithFormat:NSLocalizedString(@"Merge Commit Into \"%@\" Branch", nil), branch.name]];
                               }
                               if ([self.repository performReferenceTransformWithReason:(isBranch ? @"merge_branch" : @"merge_commit")
-                                                   argument:(isBranch ? [commitOrBranch name] : [commitOrBranch SHA1])
-                                                   error:&error
+                                                                              argument:(isBranch ? [commitOrBranch name] : [commitOrBranch SHA1])
+                                                                              error:&error
```
Comment (lucasderraugh) [inline GitUpKit/Views/GIMapViewController+Operations.m]: Confused by this style. Isn't it usually colon aligned if it's in this style? Seems inconsistent here as all the other params are colon aligned except `error`

File: GitUpKit/Core/GCFunctions.m (hunk 1 above)
Comment (zwaldowski) [reply GitUpKit/Core/GCFunctions.m]: Confused as well. Will look.

File: GitUpKit/Views/GIMapViewController+Operations.m (hunk 2 above)
Comment (zwaldowski) [reply GitUpKit/Views/GIMapViewController+Operations.m]: Confused as well. Might be related it being in a `()`. Will look

Comment (zwaldowski) [reply GitUpKit/Views/GIMapViewController+Operations.m]: Seemingly it's getting confused by the prior line's `(isBranch ? [commitOrBranch name] : [commitOrBranch SHA1])`

**Comment by zwaldowski** [comment]
I updated a few days ago, btw.

**Comment by lucasderraugh** [comment]
Ah, thanks! I’ll try to merge this in tomorrow!

**Pull request merged**